src/services/user.ts

```diff
@@ -249,7 +249,7 @@ const getBirthdate = async (): Promise<DateTime> =>
 
 const isBirthdayToday = async () => {
   const birthday = await getBirthdate();
-  return birthday?.toISODate() === DateTime.now().toISODate();
+  return birthday?.month === DateTime.now().month && birthday?.day === DateTime.now().day;
```

Comment:
I know that this is not part of this pull request, but I would love to see the birthday logic cleaned up to not use the `Luxon` library. Everything you're doing with these dates can be done really easily with native JS dates:
```typescript
const getBirthdate = async (): Promise<Date> =>
  http.get('profiles/birthdate').then(Date.parse) // Date.parse handles ISO 8601 format natively

const isBirthdayToday = async () => {
  const now = Date.now();
  const birthdate = await getBirthdate();
  return birthdate.getMonth() === now.getMonth() && birthdate.getDate() === now.getDate();
}
```